src/scss/alert/_status.scss

```diff
@@ -0,0 +1,51 @@
+@import 'src/scss/alert/status-map';
+
+
+/// Alert message status
+///
+/// Outputs status specific styles for an alert message
+/// @example
+/// @include oMessageStatus('success', 'my-class');
+///
+/// @param {String} $status - the name of the status for the message ('success', 'neutral', 'error')
+/// @param {String} $class - base class for status
+
+
+@mixin oMessageStatus ($status, $class) {
+	@if map-has-key($_o-message-status-styles, $status) {
```

Comment:
Opinion only: Exiting early rather than wrapping in if/else can avoid indention and make code a little easier to read in my opinion. This is fine though :)
```javascript
@if not map-has-key($_o-message-status-styles, $status) {
    @return _oMessageError("#{$status} is not a status. You can try 'error', 'success' or 'neutral', instead.");
}
...
```